Assets/SplineMesh/Scripts/Editor/SplineEditor.cs

```diff
@@ -277,20 +277,19 @@ public override void OnInspectorGUI()
             if (GUILayout.Button("Add node after selected"))
             {
                 Undo.RecordObject(spline, "add spline node");
-                spline.SplitAtTime(5.25f);
-
-                //SplineNode newNode = new SplineNode(selection.Direction, selection.Direction + selection.Direction - selection.Position);
-                //var index = spline.nodes.IndexOf(selection);
-                //if (index == spline.nodes.Count - 1)
-                //{
-                //    spline.AddNode(newNode);
-                //}
-                //else
-                //{
-                //    spline.InsertNode(index + 1, newNode);
-                //}
-                //selection = newNode;
-                //serializedObject.Update();
+
+                SplineNode newNode = new SplineNode(selection.Direction, selection.Direction + selection.Direction - selection.Position);
+                var index = spline.nodes.IndexOf(selection);
+                if (index == spline.nodes.Count - 1)
+                {
+                    spline.AddNode(newNode);
+                }
+                else
+                {
+                    spline.InsertNode(index + 1, newNode);
+                }
+                selection = newNode;
+                serializedObject.Update();
```

Comment:
this is the original code It was commented by mistake when testing the new method.

this commit restores it